app/__init__.py

```diff
@@ -1,9 +1,14 @@
+import sys
 from flask import Flask
 from sassutils.wsgi import SassMiddleware
 
+# Ensure that the config is always imported when starting the application
+sys.path.append('/home/gi-liam/Documents/Code/gwasdiversitymonitor/deploy')

```

Comment:
If this is going into production, we might need to use`import os`, `os.path.join(os.get_cwd(), 'deploy') or similar

Reply:
Otherwise looks good to me.

Reply:
I have updated the config path to be relative so the path can be resolved on production